src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -172,10 +172,26 @@ object Coroutine {
       $yield
     }
 
+    /** Returns whether or not the coroutine yielded a value. This value can be
+     *  accessed via `getValue`.
+     *
+     *  @return `true` if the coroutine yielded a value, `false` otherwise.
+     */
     final def hasValue: Boolean = $hasYield
 
+    /** Returns an `Option` instance wrapping the current value of the coroutine,
+     *  if any.
+     *
+     *  @return `Some(value)` if `hasValue`, `None` otherwise.
+     */
     final def getValue: Option[Y] = if (hasValue) Some(value) else None
 
+    /** Returns a `Try` instance wrapping either the current value of this coroutine
```

Comment:
Same thing here. Are you talking about separating the second sentence onto its own line or something else?